snowfakery/__init__.py

```diff
@@ -10,3 +11,35 @@
 version_file = Path(__file__).parent / "version.txt"
 with version_file.open() as f:
     version = f.read().strip()
+
+
+def generate_data(
+    yaml_file: Union[Path, str],
+    option: List[Tuple[str, str]] = [],
```

Comment:
Why is this name singular? And why is it a list instead of a dict?

Reply:
I was translating from the click-derived API too literally. Fixed now. "user_options".

4119821